docs/data/rpc/admin-guide/configuring.mdx

```diff
@@ -0,0 +1,332 @@
+
+### Build From Source
+
+Instructions for building Stellar RPC from source can be found [here](https://github.com/stellar/stellar-rpc/blob/main/cmd/stellar-rpc/README.md).
+
+## Use your Local RPC Instance
+
+You can configure Stellar CLI to use your local remote RPC endpoint:
+
+```bash
+stellar network add --global local \
+    --rpc-url http://localhost:8000 \
+    --network-passphrase 'Standalone Network ; February 2017'
+```
+
+And fund your accounts/identities with the Friendbot:
+
+```bash
+curl "http://localhost:8000/friendbot/?addr=$(stellar keys address alice)"
+```
+
+See the tip above about command expansion (that's the note about `$(...)`) if you're not using a bash-based shell.
+
+:::caution
+
+When interacting with the network in production you should run your own stellar-rpc or use a third-party infrastructure provider.
+
+:::
+
+## Verify the RPC Instance
+
+After installation, it will be worthwhile to verify the installation of Stellar RPC is healthy. There are two methods:
+
+1. Access the health status endpoint of the JSON RPC service using an HTTP client
+2. Use our pre-built [System Test Docker image](https://hub.docker.com/r/stellar/system-test/tags) as a tool to run a set of live tests against Stellar RPC
+
+### Health Status Endpoint
+
+If you send a JSON RPC HTTP request to your running instance of Stellar RPC:
+
+```bash
+curl --location 'http://localhost:8000' \
+--header 'Content-Type: application/json' \
+--data '{
+"jsonrpc":"2.0",
+"id":2,
+"method":"getHealth"
+}'
+```
+
+You should get back an HTTP 200 status response if the instance is healthy:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 2,
+  "result": {
+    "status": "healthy"
+  }
+}
+```
+
+### System Test Docker Image
+
+This test will compile, deploy, and invoke example contracts to the network to which your RPC instance is connected to. Here is an example for verifying your instance if it is connected to Testnet:
+
+```bash
+docker run --rm -t --name e2e_test \
+docker.io/stellar/system-test:soroban-v20 \
+--VerboseOutput true \
+--TargetNetworkRPCURL <your rpc url here> \
+--TargetNetworkPassphrase "Test SDF Network ; September 2015" \
+--TargetNetworkTestAccountSecret <your test account StrKey encoded key here > \
+--TargetNetworkTestAccountPublic <your test account StrKey encoded pubkey here> \
+--SorobanExamplesGitHash v22.0.1
+```
+
+Make sure you configure the system test correctly:
+
+- Use a published [system test tag](https://hub.docker.com/r/stellar/system-test/tags) that aligns with the deployed Stellar RPC release version. For each release, there will be two tags published for different CPU architectures. Make sure to use the tag that matches your host machine's CPU architecture:
+- `docker.io/stellar/system-test:<release name>` for machines with AMD/Intel-based CPUs
+- `docker.io/stellar/system-test:<release name>-arm64` for machines with ARM CPUs
```

Comment:
```suggestion
```

per the [suggestion](https://github.com/stellar/stellar-docs/pull/1392/files#r1990055398) to build system test locally, can delete lines 320-322 as they refer to using pre-built images which isn't viable for now.